[core]/es_extended/client/modules/points.lua

```diff
@@ -0,0 +1,60 @@
+local Markers = {}
+
+---@func fun(coords: vector3, radius: number, data?: any)
+function ESX.CreateMarker(data) 
+    local Index = #Points+1
+    Markers[Index] = data
+    return Markers[Index]
+end
+
+CreateThread(function()
+    while true do 
+        Wait(0)
+        local InPoint = false
```

Comment:
Change all variable is correct lowerCamelCase. Thanks

Reply:
Removed Variable for a better way to do what it was meant for